lib/network.sh

```diff
@@ -18,6 +18,15 @@ function network_address() {
   export resultvar
 }
 
+# REF: https://forum.archive.openwrt.org/viewtopic.php?id=47986&p=1#p220781
+function cdr2mask ()
+{
+   # Number of args to shift, 255..255, first non-255 byte, zeroes
+   set -- $(( 5 - ($1 / 8) )) 255 255 255 255 $(( (255 << (8 - ($1 % 8))) & 255 )) 0 0 0
```

Comment:
This won't work with ipv6 - I think a better approach would be to use python, e.g similar to:

https://github.com/metal3-io/metal3-dev-env/blob/master/lib/network.sh#L41

Reply:
Ohh. Didn't realize that. Changed it to python. PTAL